src/main/java/com/microsoft/azure/functions/worker/handler/PicoHelper.java

```diff
@@ -0,0 +1,14 @@
+package com.microsoft.azure.functions.worker.handler;
+
+import com.sun.jna.win32.StdCallLibrary;
+import com.sun.jna.Native;
+import com.sun.jna.win32.W32APIOptions;
+
+// kernel32.dll uses the __stdcall calling convention (check the function
+// declaration for "WINAPI" or "PASCAL"), so extend StdCallLibrary
+// Most C libraries will just extend com.sun.jna.Library,
+public interface PicoHelper extends StdCallLibrary { 
+    PicoHelper INSTANCE = (PicoHelper)Native.load("picohelper", PicoHelper.class, W32APIOptions.DEFAULT_OPTIONS);
```

Comment:
@safihamid - Pushed changes to fix build errors. Unit test is failing with error
```
java.lang.UnsatisfiedLinkError: Unable to load library 'picohelper': Native library (win32-x86-64/picohelper.dll) not found in resource path 
```
missed adding picphelper.dll?

Reply:
Thanks. Will update the test.